controller/validateUserController.js

```diff
@@ -6,19 +6,42 @@ config();
 const validateUserController = (req, res) =>{
   const authorization = req.headers["authorization"].split(" ");
   jwt.verify(authorization[1], process.env.JWT_KEY, (err, decoded) => {
```

Comment:
get the value of `jwt.verify()` directly

Reply:
The `jwt.verify()` method returns a payload, so you can get that payload directly like so 👇
```
const payload = jwt.verify(authorization[1], process.env.JWT_KEY)

if (!payload.data.id) {  //then check if the payload has the required properties, if not return an error status code 👇
   return res.sendStatus(401)
}

//then destructure the propertiies from the payload
const {id, password} = payload.data

// and then do the rest logic from there
```
This way you can reduce the amount of nested callbacks.

Reply:
The Paylod is of this nature 
```
{
  data: {
    id: '<some id string>',     
    email: '<some email string>',
    password: '<some password string>'
  },
  iat: 1662950425
}
```